tests/publiccloud/upload_image.pm

```diff
@@ -34,7 +34,15 @@ sub run {
         return;
     }
 
-    assert_script_run("wget --no-check-certificate $img_url -O $img_name", timeout => 60 * 10);
+    # Download the given image. Check for 404 errors and make them better visible
+    my $cmd = "wget --no-check-certificate $img_url -O $img_name";
+    my $rc  = script_run("$cmd 2>download.txt", timeout => 60 * 10);
+    if ($rc != 0) {
+        upload_logs("download.txt");
```

Comment:
not sure how it could help . you already has exactly same info here http://duck-norris.qam.suse.de/tests/6748/logfile?filename=serial_terminal.txt

Reply:
Instead of a "`wget` died and please dive into the serial terminal to investigate the issue" you see immediately the actual error.
This is helpful when reviewing a whole lot of jobs and at some point one gets tired of browsing through the serial output.

It is not important for a single job where you have a close look at. It is helpful when reviewing a whole bunch of jobs and you don't want to waste a lot of time scrolling though the serial output of every job. Highlighting common errors like this one saves us a bunch of time, nerves and false positives.